forgettable-frontend/src/components/EncounterCardSummary/EncounterSummary.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import {Avatar} from '@mui/material';
+import './EncounterSummary.css';
+
+export default function EncounterSummary(props: { Name: any; Date: any; Description: any; Summary: any; }) {
+  const {
+    Name, Date, Description, Summary,
+  } = props;
+
+  return (
+    <div className="container">
+      <Card
+        className="card"
+        sx={{maxWidth: 220, maxHeight: 240, bgcolor: 'var(--lcard)', borderRadius: 6}}
```

Comment:
You should be able to put this styling CSS inside your module.css file as well, so can leave the js file without any stylings

Reply:
I tried doing that, but the css changes weren't overriding the default MUI styles, I can look into how to do it properly further tho.

Reply:
Ohh right, if it's difficult we can just leave it like that for now!